connector/connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -170,6 +170,8 @@ message Expression {
 
   message Alias {
     Expression expr = 1;
-    string name = 2;
+    repeated string name = 2;
```

Comment:
In catalyst, we have a `MultiAlias` expression for this case. Shall we match catalyst here?

Reply:
Since we're not trying to match all of the Catalyst plans but be compatible on the logical level, I find it weird to use `MultiAlias` here.

Reply:
OK, then shall we fail if it's a multi alias but the metadata fied is specified?

Reply:
makes sense, let me add a server side check to the client side one

Reply:
done